### PR TITLE
Created the 2019-education package

### LINKED ISSUE #667: Create 2019-education package
### What

The `2019-education` package needs to be created, based on the `2019-template` package, and added as a hidden route. See #634 for more information.

### Why

Part of building 6 amazing projects on CIVIC

### When

ASAP

### PR DESCRIPTION
**Pull request by jaikamat**
Closes #667 by creating the `2019-education` package. 

- Copied template directory and edited references from `template` to `education`
- Included sample dataset and simple card
- Removed seemingly redundant `demo-card` and `demo-data`

### CODE REVIEW

File: packages/2019-education/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## 2019 Education Project
+
+Provides assets for the Education project for the 2019 Hack Oregon project season.
+
+Designed to be copied when creating new projects for the 2019 Hack Oregon project season.
+
+### When initializing a new project:
```
Comment (Dianna) [inline packages/2019-education/README.md]: Can remove this copy pasta section

**Comment by Dianna** [comment]
This is awesome work and you did it exactly right, however... we just merged this really great script which will generate all of this AND instead of relying on local data or dealing with difficult api connections it will do it for you. So what I'm proposing is that you can use our [new method here](https://github.com/hackoregon/civic#creating-a-new-card), specifically the [using api data section](https://github.com/hackoregon/civic#using-api-data-recommended) which will make your future work easier. The [local data section](https://github.com/hackoregon/civic#using-local-data-not-recommended) is basically what you've done here.

**Review by ghost — COMMENTED** [review]
This is awesome work and you did it exactly right, however... we just merged this really great script which will generate all of this AND instead of relying on local data or dealing with difficult api connections it will do it for you. So what I'm proposing is that you can use our [new method here](https://github.com/hackoregon/civic#creating-a-new-card), specifically the [using api data section](https://github.com/hackoregon/civic#using-api-data-recommended) which will make your future work easier. The [local data section](https://github.com/hackoregon/civic#using-local-data-not-recommended) is basically what you've done here.

**Comment by jaikamat** [comment]
@Dianna Nice, new script sounds super exciting! Should I go ahead and make a new PR with these two new packages using the new script?

**Comment by Dianna** [comment]
@jaikamat I would say yes! Let me know if you run into any issues

**Comment by jaronheard** [comment]
@jaikamat -- hey, we made a lot of changes to how the new card script works and it got a bit messy in the middle, so we were going to have @Dianna wrap this up, thanks for doing this! You totally did the work on this, we just changed the world under you!

**Comment by jaikamat** [comment]
@jaronheard No worries Jaron! Thanks for the heads up!

**Comment by Dianna** [comment]
Superseded by #774

**Pull request closed without merging**